app/containers/CompanyReview/reducer.js

```diff
@@ -0,0 +1,29 @@
+import { fromJS } from 'immutable';
+
+import {
+  ADD_REVIEW_SUCCESS,
+  ADD_REVIEW_FAILURE,
+  SET_HAS_REVIEWED,
+} from './constants';
+
+const initialState = fromJS({
+  errorMessage: '',
+  hasReviewed: false,
+  canComment: false,
```

Comment:
This shouldn't be in redux. We talked about this and agreed to keep this information in the component state.